Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveE.cpp

```diff
@@ -174,10 +179,9 @@ void FiniteDifferenceSolver::MacroscopicEvolveECartesian (
         // Loop over the cells and update the fields
         amrex::ParallelFor(tex, tey, tez,
             [=] AMREX_GPU_DEVICE (int i, int j, int k){
-#ifdef AMREX_USE_EB
                 // Skip field push if this cell is fully covered by embedded boundaries
-                if (lx(i, j, k) <= 0) return;
-#endif
+                if (lx && lx(i, j, k) <= 0) { return; }
```

Comment:
Same comment as further up: we could rename this as `eb_lx` to make it clear that it is related to the embedded boundary.